common/junit-platform-native/build.gradle

```diff
@@ -1,85 +1,81 @@
+/*
```

Comment:
If the entire project is now using Kotlin, should we move away from Groovy here as well?
(just a question - we can do it in another PR in order to reduce scope)

Reply:
I don't think we have to. It's perfectly fine to have a mix of Groovy and Kotlin scripts. I used Kotlin for the new ones to get IDE support.